jazkarta/shop/stripe.py

```diff
@@ -13,29 +13,17 @@ def stripe_amount(dollars):
     return int(dollars * 100)
 
 
-def call_stripe(method, url, params):
+def call_stripe():
     if config.IN_PRODUCTION:
         api_key_setting = 'stripe_api_key_production'
     else:
         api_key_setting = 'stripe_api_key_dev'
-    api_key = get_setting(api_key_setting)
-    api = stripe.APIRequestor(api_key)
-    try:
-        return api.request(method, url, params)[0]
-    except stripe.StripeError as e:
-        errmsg = e.json_body['error']['message']
-        raise PaymentProcessingException(errmsg)
-    except Exception as e:
-        raise PaymentProcessingException(str(e))
-
-
-def process_refund(charge_id, refund_amount):
```

Comment:
Was this just unused code? Don't we still need some mechanism for handling refund requests?